kustomize/commands/localize/localize.go

```diff
@@ -27,7 +26,6 @@ type flags struct {
 
 // NewCmdLocalize returns a new localize command.
 func NewCmdLocalize(fs filesys.FileSystem, writer io.Writer) *cobra.Command {
```

Comment:
super nit: We can replace "writer" with underscore "_" since we no longer use this argument.